tests/component-plugin/src/main/java/org/mule/functional/api/component/ResponseAssertionMessageProcessor.java

```diff
@@ -138,7 +138,7 @@ public void verify() throws InterruptedException {
     } else if (responseCount > 0 && responseSameTask) {
       assertThat(failureMessagePrefix() + "Response task was not same as request task", responseTaskToken,
                  is(requestTaskToken));
-    } else if (responseCount > 0 && !responseSameTask) {
```

Comment:
why remove that condition?

Reply:
It's redundant because of the previous one.